app-i18n/fbterm/fbterm-1.7_p20190503.ebuild

```diff
@@ -22,6 +22,7 @@ DEPEND="media-libs/fontconfig
 	gpm? ( sys-libs/gpm )
 	video_cards_vesa? ( dev-libs/libx86 )"
 BDEPEND="virtual/pkgconfig"
+RDEPEND="${DEPEND}"
```

Comment:
I would suggest to define variables in this order:
```
BDEPEND
DEPEND
RDEPEND
PDEPEND (Not here)
```

Reply:
Done.